manifests/jetty.pp

```diff
@@ -50,26 +52,30 @@
   file { '/var/log/jetty':
     ensure => 'link',
     target => "${jetty_home}/jetty/logs",
-  } ->
-
-  file_line { 'jetty_startup_minutes':
-    ensure => present,
-    path   => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
-    line   => "    sleep ${jetty_start_interval}",
-    match  => '^    sleep \d+',
-  } ->
-
-  file { '/etc/init.d/jetty':
-    ensure => 'link',
-    target => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
   }
 
   if $::service_provider == 'systemd' {
     systemd::unit_file { 'jetty.service':
       content => template("${module_name}/jetty/jetty.service.erb"),
-      require => File['/etc/init.d/jetty'],
       before  => Service['jetty'],
     }
+  } else {
+    file_line { 'jetty_startup_minutes':
+      ensure => present,
+      path   => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
+      line   => "    sleep ${jetty_start_interval}",
+      match  => '^    sleep \d+',
+    }
+
+    file { '/etc/init.d/jetty':
+      ensure => 'link',
+      target => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
+    }
+
+    file { '/etc/default/jetty':
+      ensure  => file,
+      content => template("${module_name}/default.erb"),
+    }
```

Comment:
These three items need some ordering with -> or require and need to be tied to the service with a notify or require.
Maybe this on the file_line:
```puppet
require => File['/var/log/jetty']
```

Then arrows to the other files, with the last file notifying the service